docs/examples/coreos-kops-tests-multimaster.md

```diff
@@ -196,7 +196,7 @@ If you don't want KOPS to auto-select the instance type, you can use the followi
 
 But, before doing that, always ensure the instance types are available on your desired AZ.
 
-NOTE: More arguments and kops commands are described on the following document: [commands.md](https://github.com/kubernetes/kops/blob/master/docs/commands.md)
+NOTE: More arguments and kops commands are described on the following document: [cli](https://github.com/kubernetes/kops/tree/master/docs/cli)
```

Comment:
One more thing, perhaps we can use a relative path here? that way it would work with both reading the markdown files on github as well as the [docs site](https://kops.sigs.k8s.io/cli/kops/)

```suggestion
NOTE: More arguments and kops commands are described [here](../cli/kops.md).
```

Reply:
@rifelpet agree 
@yuxiaobo96  update it?Thanks